### ISSUE TITLE
Decoupling exploration bar and voicing announcements

### ISSUE THREAD

**Issue by Charlotte-McCleary** [fizzstudio/ParaCharts#1206]
Currently, both self‑voicing and screen readers immediately announce anything that goes into the exploration bar. Ideally, they should be decoupled so we can choose when to send announcements to screen readers / self-voicing, and allow messages to appear in the exploration bar without triggering an automatic announcement if we want.

For example:  
Right now, the only way to have sonification play first, followed by the spoken announcement, is to delay sending the announcement text to the exploration bar. This forces the message itself to appear late. A better approach would allow the message to appear in the exploration bar immediately while independently delaying the self‑voicing / screen reader announcement so it doesn’t overlap with the sonification.
